aspnetcore/signalr/hubs/sample/Hubs/ChatHub.cs

```diff
@@ -19,8 +19,7 @@ public Task SendMessageToCaller(string message)
 
         public Task SendMessageToGroups(string message)
         {
-            List<string> groups = new List<string>() { "SignalR Users" };
-            return Clients.Groups(groups).SendAsync("ReceiveMessage", message);
+            return Clients.Group("SignalR Users").SendAsync("ReceiveMessage", message);
```

Comment:
Would updating this change the meaning of the docs for `Send messages to clients`?  I don't think it would - but I pushed up the change before thinking about it.